buildSrc/src/main/java/Config.kt

```diff
@@ -11,7 +11,7 @@ object Config {
         private val sdkVersion = 29
 
         val buildToolsVersion = "29.0.2"
-        val minSdkVersion = 14
+        val minSdkVersion = 21
```

Comment:
I'd not do it, set version 21 directly then instead

Reply:
I was hoping we'd figure out a way to keep this on 14 when possible. Perhaps only NDK support means requiring 21?
We're not requiring 21+ across the board, are we?

Reply:
actually the client App. overrides it, but this is important to our static code analysis

Reply:
I didn't investigate it deeper yet, but I'd say that if we need to set API up to 21, we might be using ndk features that were not retro compatible and we would need to do some workarounds if we wanna fully support older versions.
ps: https://android.googlesource.com/platform/ndk/+/master/docs/user/common_problems.md#Target-API-Set-Higher-Than-Device-API

Reply:
also found this one https://android.googlesource.com/platform/ndk/+/master/docs/BuildSystemMaintainers.md
which says:

> For example, applications with a minSdkVersion of 20 should use API 19 for their NDK target.
it means that if we wanna support v14, we should set v13